### PR TITLE
Add cmake support

### PR DESCRIPTION
**Pull request by Hofer-Julian**

### CODE REVIEW

File: src/terminal.ts [hunk 1]
```diff
@@ -166,80 +166,160 @@ export class FlatpakManifest {
...
-          })
-        }
-        break
+        return this.getSimpleCommands(module['build-commands'], buildArgs)
+      case 'qmake':
+        throw new Error('Qmake is not implemented yet')
+    }
+    throw new Error('Failed to build application')
+  }
+  getCmakeCommands(rebuild: boolean, buildArgs: string[], configOpts: string): Command[] {
```
Comment (bilelmoussaoui) [inline src/terminal.ts:185]: I know that I didn't do that for meson, but we should definitely have some comment about which commands we execute inside the sandbox. So something like
```
The CMake build system builds the module using the following commands:
-`mkdir -p _build`

.....
```

Comment (bilelmoussaoui) [inline src/terminal.ts:185]: Also nitpick: could use an empty line between the two functions

Comment (Hofer-Julian) [reply src/terminal.ts:197]: Done

Comment (Hofer-Julian) [reply src/terminal.ts:197]: Done

**Comment by bilelmoussaoui** [comment]
Nice, thanks!

**Pull request merged**